src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/LdapDirectoryDAO.scala

```diff
@@ -255,12 +255,40 @@ class LdapDirectoryDAO(
     }
   }
 
-  override def listIntersectionGroupUsers(groupIds: Set[WorkbenchGroupIdentity]): Future[Set[WorkbenchUserId]] = Future {
-    ldapSearchStream(
-      directoryConfig.baseDn,
-      SearchScope.SUB,
-      Filter.createANDFilter(groupIds.map(groupId => Filter.createEqualityFilter(Attr.memberOf, groupDn(groupId))).asJava)
-    )(getAttribute(_, Attr.uid)).flatten.map(WorkbenchUserId).toSet
+//  override def listIntersectionGroupUsers(groupIds: Set[WorkbenchGroupIdentity]): IO[Set[WorkbenchUserId]] = IO {
+//    ldapSearchStream(
+//      directoryConfig.baseDn,
+//      SearchScope.SUB,
+//      Filter.createANDFilter(groupIds.map(groupId => Filter.createEqualityFilter(Attr.memberOf, groupDn(groupId))).asJava)
+//    )(getAttribute(_, Attr.uid)).flatten.map(WorkbenchUserId).toSet
+//  }
```

Comment:
this was the big bad isMemberOf query... I am not ready to forget it so I am leaving it in for now